--- FtcRobotControllerActivity.java.orig	2016-11-23 17:12:41.877405678 -0500
+++ FtcRobotControllerActivity.java	2016-11-23 17:09:08.276294267 -0500
@@ -91,6 +91,8 @@
 import org.firstinspires.ftc.robotcore.internal.AppUtil;
 import org.firstinspires.inspection.RcInspectionActivity;
 
+import io.github.aedancullen.triangular.VisionProcessor;
+
 import java.io.File;
 import java.util.Queue;
 import java.util.concurrent.ConcurrentLinkedQueue;
